src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -96,6 +96,9 @@ public int Execute()
 
             DisplayTestFiles();
 
+			// Throw a FileNotFoundException if any input file is missing.
+			CheckExistenceOfInputFiles(_options);
+
```

Comment:
As explained above, we don't want to do this. In fact, this is very much like the code we removed from NUnit 2 because we wanted to run any files we could run even if one turns out to be bad.